### PR TITLE
Further polishing

### PR DESCRIPTION
**Pull request by pnrobinson**
This PR adds the GNU3 licence, adapts the Retinoblastoma example to use string instead of any, and uses the "of" and "builder" names for static functions that return a finished object or a builder. The one exception is ontologyClass which is used so often it is good to make an exception. Otherwise, using this pattern is clearer (i.e., I agree with your suggestion).

### CODE REVIEW

**Comment by julesjacobsen** [comment]
Please don't use GPL-3 it's really *not* what we want. BSD-3 or MIT would be far more appropriate.

**Comment by pnrobinson** [comment]
Um, what don;t you like about GNU3? I am a fan of sorts because of the open source virality.

**Comment by julesjacobsen** [comment]
https://github.com/phenopackets/phenopacket-tools/issues/53#issuecomment-1102879268

**Comment by julesjacobsen** [comment]
Just found the GA4GH doc on this: https://www.ga4gh.org/wp-content/uploads/GA4GH-Copyright-Policy-Updated-Formatting.pdf

TLDR - Use Apache 2.0
